showcase/app/templates/components/copy/button.hbs

```diff
@@ -142,6 +142,17 @@
     </SF.Item>
   </Shw::Flex>
 
+  <Shw::Text::H4>Edge cases</Shw::Text::H4>
```

Comment:
[suggestion] At this point, they're not "technically" edge, more of special values that we have to handle correctly

```suggestion
  <Shw::Text::H4>Special cases</Shw::Text::H4>
```